apollo-client/src/test/java/com/ctrip/framework/apollo/internals/RemoteConfigLongPollServiceTest.java

```diff
@@ -476,6 +475,11 @@ public String getCluster() {
       return someCluster;
     }
 
+    @Override
+    public String getAccessKeySecret() {
+      return someSecret;
```

Comment:
shall we add some tests to test whether secret is in the http request header?

And I think it's better to add different scenarios to test when the secret is disabled, which is the default behavior.